react/features/filmstrip/subscriber.native.js

```diff
@@ -1,25 +1,17 @@
 // @flow
 
-import { getParticipantCountWithFake } from '../base/participants';
 import { StateListenerRegistry } from '../base/redux';
 import { shouldDisplayTileView } from '../video-layout';
 
 import { setTileViewDimensions } from './actions';
+import { getTileViewParticipantCount } from './functions.native';
 import './subscriber.any';
 
 /**
  * Listens for changes in the number of participants to calculate the dimensions of the tile view grid and the tiles.
  */
 StateListenerRegistry.register(
-    /* selector */ state => {
-        const participantCount = getParticipantCountWithFake(state);
-
-        if (participantCount < 6) { // the dimensions are updated only when the participant count is lower than 6.
```

Comment:
Why are we undoing this?

Reply:
In order to be able to set `hasScroll` flag.